examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -89,7 +89,14 @@ CommissioningParameters PairingCommand::GetCommissioningParameters()
 
 CHIP_ERROR PairingCommand::PaseWithCode(NodeId remoteId)
 {
-    return CurrentCommissioner().EstablishPASEConnection(remoteId, mOnboardingPayload);
+    chip::SetupPayload payload;
+
+    QRCodeSetupPayloadParser(mOnboardingPayload).populatePayload(payload);
+    mSetupPINCode  = payload.setUpPINCode;
+    mDiscriminator = payload.discriminator;
+
+    RendezvousParameters rendezvousParams = RendezvousParameters().SetSetupPINCode(mSetupPINCode).SetDiscriminator(mDiscriminator);
+    return CurrentCommissioner().EstablishPASEConnection(remoteId, rendezvousParams);
```

Comment:
This still looks like it will break things.  The point is that the version of `EstablishPASEConnection` that takes params assumes that discovery has already happened and the params have the right address to use.

Why are we changing this function at all?